modules/rules_foreign_cc/0.8.0/MODULE.bazel

```diff
@@ -0,0 +1,35 @@
+"bazelbuild/rules_foreign_cc"
+
+module(
+    name = "rules_foreign_cc",
+    version = "0.8.0",
+    compatibility_level = 1,
+    toolchains_to_register = [
+        "@rules_foreign_cc_framework_toolchain_freebsd//:toolchain",
+        "@rules_foreign_cc_framework_toolchain_linux//:toolchain",
+        "@rules_foreign_cc_framework_toolchain_macos//:toolchain",
+        "@rules_foreign_cc_framework_toolchain_windows//:toolchain",
+        "@rules_foreign_cc//toolchains:built_make_toolchain",
```

Comment:
FYI: You can use repo-relative labels in `MODULE.bazel`, so this doesn't need the `@rules_foreign_cc`.